scripts/build-runtime-srtool.sh

```diff
@@ -13,8 +13,8 @@ CMD="docker run \
   -e CARGO_NET_GIT_FETCH_WITH_CLI=true \
   -e PACKAGE=${GH_WORKFLOW_MATRIX_CHAIN}-runtime \
   -e RUNTIME_DIR=runtime/${GH_WORKFLOW_MATRIX_CHAIN} \
-  -e BUILD_OPTS=--features on-chain-release-build \
-  -e PROFILE=production \
+  -e BUILD_OPTS=${RUNTIME_BUILD_OPTS} \
```

Comment:
```suggestion
  -e BUILD_OPTS='${RUNTIME_BUILD_OPTS}' \
```

Reply:
testing locally that didn't work for me, but using the variable we can include the quotes in the workflow definition